lib/install/config/babel.config.js

```diff
@@ -28,44 +28,31 @@ module.exports = function(api) {
       (isProductionEnv || isDevelopmentEnv) && [
         '@babel/preset-env',
         {
-          forceAllTransforms: true,
           useBuiltIns: 'entry',
           corejs: 3,
           modules: false,
+          bugfixes: true,
+          loose: true,
           exclude: ['transform-typeof-symbol']
         }
       ]
     ].filter(Boolean),
     plugins: [
       'babel-plugin-macros',
-      '@babel/plugin-syntax-dynamic-import',
       isTestEnv && 'babel-plugin-dynamic-import-node',
```

Comment:
This should also be able to be removed, see https://github.com/facebook/create-react-app/pull/8353 and https://babeljs.io/blog/2020/01/11/7.8.0 and